server/controllers/userController.js

```diff
@@ -104,7 +104,23 @@ const updateUser = async (req, res) => {
       { $set: req.body },
       { new: true }
     );
-    console.log(user);
+    res.json(user);
+  } catch (error) {
+    throw error;
+  }
+};
+
+// @desc    Add company to user companies
+// @route   PATCH /api/users/addcompany
+// @access  Private
+const addCompany = async (req, res) => {
+  try {
+    const user = await User.findByIdAndUpdate(
+      req.user._id,
+      { $push: req.body },
```

Comment:
I can't find any documentation on this $push. Does it just push to the companies array the new company?

Reply:
Yeah, It just appends one company to the companies array.

Reply:
check here: https://docs.mongodb.com/manual/reference/operator/update/push/